guiEditor/src/guiNodeTools.ts

```diff
@@ -40,11 +46,42 @@ export class GUINodeTools {
                 element = new TextBlock("Textblock");
                 element.text = "My Text";
                 return element;
+            case "ImageButton":
+                element = Button.CreateImageButton("Button", "Click Me", "textures/grass.png");
+                break;
+            case "VirtualKeyboard":
+                element = new VirtualKeyboard();
+                element.addKeysRow(["1","2", "3","\u2190"]);
+                break;
+            case "Image": 
+                element = new Image("Image", "https://i.imgur.com/ntIgFT6.jpg");
```

Comment:
Are the pictures on imgur available forever? I guess you should instead point to a local Babylon texture, as you did for the image button.